tracee-rules/signatures/golang/code_injection_test.go

```diff
@@ -183,3 +186,19 @@ func TestCodeInjection(t *testing.T) {
 		}
 	}
 }
+
+func TestCodeInjection_GetMetadata(t *testing.T) {
```

Comment:
are you sure we want to test this function? it doesn't contain any code, just a declaration. if we wanted to removed the error returned (discussed in meeting), then it's an indication that there's nothing here to test.

Reply:
You're right, it doesn't do anything today. But having said that it may change tomorrow in which case a test would ensure behaviour correctness. Speaking of which I'm now leaning more towards keeping the error in the signature.